src/app/cli/tests/tests/test_datafusion_cli.rs

```diff
@@ -0,0 +1,34 @@
+// Copyright Kamu Data, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+/////////////////////////////////////////////////////////////////////////////////////////
+
+use assert_cmd::Command;
+
+#[test]
+fn test_datafusion_cli() {
+    let expected_output =
+        "+----------+\n| Int64(1) |\n+----------+\n| 1        |\n+----------+\n1 row in set.";
+
+    let mut cmd = Command::cargo_bin("kamu-cli").unwrap();
+
+    let assert = cmd.arg("sql").write_stdin("select 1;").assert();
+
+    let output = assert.get_output();
+
+    //not using assert.stdout("expected text") as time of query execution may
+    // differ
+    let string_result = String::from_utf8_lossy(&output.stdout);
+
+    assert!(
+        string_result.contains(expected_output),
+        "Output does not contain expected text"
+    );
```

Comment:
Let's change this to:
```rust
assert!(
  string_result.contains(expected_output),
  "Output does not contain expected text: {}",
  string_result
);
```
to see why these tests are failing in CI.

Reply:
changed tests => tests have been failing as command fails to be executed. Output is empty.

Reply:
Try printing out STDERR of the process before asserting on status.

Also I would try locally to do `cargo clean` and then run the tests using same commands as our `build` CI action uses. My intuition is that perhaps these tests are running before the `kamu-cli` binary was compiled and your local tests are passing because you already have the binary from the previous compilation.

Reply:
Hey @sergiimk! Thanks for helping! I have tried: clean and used cargo nextest run --cargo-profile ci -E '!test(::containerized::)' to run test on mac:   Summary [   5.112s] 431 tests run: 431 passed, 85 skipped
Worked as a charm. I also tried to execute yesterday just --help command and all tests passed for all envs. That's something with sql command. I am on it.